### PR TITLE
feat: download evaluation dataset

### PR DESCRIPTION
**Pull request by axiomofjoy**
I don't currently have access to `phoenix-assets`. Writing JSONL files to cloud storage for now.

### CODE REVIEW

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/experimental/evals/utils.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        task (str): Task to be performed.
+        dataset_name (str): Name of the dataset.
+
+    Returns:
+        pandas.DataFrame: A pandas dataframe containing the data.
+    """
+    url = f"http://storage.googleapis.com/arize-assets/phoenix/evals/{task}/{dataset_name}.jsonl"
+    try:
+        with urlopen(url) as f:
+            return pd.DataFrame([json.loads(line.decode()) for line in f.readlines()])
```
Comment (RogerHYang) [inline src/phoenix/experimental/evals/utils.py:25]: what does `decode` do?
```suggestion
            return pd.DataFrame(map(json.loads, f.readlines()))
```

Comment (RogerHYang) [reply src/phoenix/experimental/evals/utils.py:25]: also, these files are so large it may worth considering storing them as zipped files

Comment (axiomofjoy) [reply src/phoenix/experimental/evals/utils.py:25]: Good catch, `decode` was unnecessary here.

Comment (axiomofjoy) [reply src/phoenix/experimental/evals/utils.py]: addressed in [9eadbdc](https://github.com/Arize-ai/phoenix/pull/1241/commits/9eadbdce17df172747c37792a513e10a41908ee4)

File: src/phoenix/experimental/evals/utils.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import json
+from io import BytesIO
+from urllib.error import HTTPError
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import pandas as pd
+
+
+def download_benchmark_dataset(task: str, dataset_name: str) -> pd.DataFrame:
```
Comment (RogerHYang) [inline src/phoenix/experimental/evals/utils.py:14]: just an idea: these two arguments can be combined as one e.g. "task/dataset_name" and that would not affect the http address

Comment (axiomofjoy) [reply src/phoenix/experimental/evals/utils.py:14]: Oh shoot, missed this comment, but that's a good idea.

**Pull request merged**